Source/WebKit/GPUProcess/media/RemoteMediaDescription.h

```diff
@@ -40,21 +40,21 @@ class RemoteMediaDescription : public WebCore::MediaDescription {
 
     virtual ~RemoteMediaDescription() = default;
 
-    AtomString codec() const final { return m_codec; }
+    StringView codec() const final { return m_codec; }
     bool isVideo() const final { return m_isVideo; }
     bool isAudio() const final { return m_isAudio; }
     bool isText() const final { return m_isText;}
 
 private:
     RemoteMediaDescription(const MediaDescriptionInfo& descriptionInfo)
-        : m_codec(descriptionInfo.m_codec)
+        : m_codec(descriptionInfo.m_codec.isolatedCopy())
         , m_isVideo(descriptionInfo.m_isVideo)
         , m_isAudio(descriptionInfo.m_isAudio)
         , m_isText(descriptionInfo.m_isText)
     {
     }
 
-    AtomString m_codec;
+    String m_codec;
```

Comment:
const

Reply:
ok, I've refactored to make m_codec a member of the base class.